components/src/molecules/Tabs.tsx

```diff
@@ -0,0 +1,80 @@
+import * as React from 'react'
+import { css } from 'styled-components'
+import { TYPOGRAPHY, SPACING } from '../ui-style-constants'
+import { COLORS, BORDERS } from '../helix-design-system'
+import { POSITION_RELATIVE, DIRECTION_COLUMN, DIRECTION_ROW } from '../styles'
+import { Flex } from '../primitives'
+
+const defaultTabStyle = css`
+  ${TYPOGRAPHY.pSemiBold}
+  color: ${COLORS.black90};
+  background-color: ${COLORS.purple30};
+  border: 0px ${BORDERS.styleSolid} ${COLORS.purple30};
+  border-radius: ${BORDERS.borderRadius8};
+  padding: ${SPACING.spacing8} ${SPACING.spacing16};
+  position: ${POSITION_RELATIVE};
+
+  &:hover {
+    background-color: ${COLORS.purple35};
+  }
+
+  &:focus-visible {
+    outline-offset: 2px;
+    outline: 2px ${BORDERS.styleSolid} ${COLORS.blue50};
+  }
+
+  &:disabled {
+    background-color: ${COLORS.grey30};
+    color: ${COLORS.grey40};
+  }
+`
+const currentTabStyle = css`
+  ${defaultTabStyle}
+  color: ${COLORS.white};
+  background-color: ${COLORS.purple50};
+
+  &:hover {
+    background-color: ${COLORS.purple55};
+  }
+`
+
+export interface TabsProps {
+    buttons: Array<{
+        text: string
+        isActive?: boolean
+        disabled?: boolean
+        onClick: () => void
```

Comment:
nit: optional props should go last

```suggestion
        text: string
        onClick: () => void
        isActive?: boolean
        disabled?: boolean
```